liferay-content-management/liferay-portal
# LPD-15146 Adapt headless methods with new fields

# What is this trying to solve?

-https://liferay.atlassian.net/browse/LPD-15146 Adapt headless methods with new fields

# How am I fixing it?

Modifying the document API definition to incorporate these new fields.

# How can you verify that it works?
  1. Create a content assigning it a publication date and an expiration date.
  2. Retrieve the content using the API GUI.
  3. Check that the new fields are populated.

@manuelRives, @robertoDiaz
